src/shared/treeview/nodes/errorNode.ts

```diff
@@ -5,13 +5,30 @@
 
 import * as vscode from 'vscode'
 import { AWSTreeNodeBase } from './awsTreeNodeBase'
+import { localize } from '../../../shared/utilities/vsCodeUtils'
+import { getLogger, Logger } from '../../../shared/logger/logger'
+import { getLogTracker, LogTrackerRecord } from '../../../shared/logger/logTracker'
 
 // Used as a child node when an exception occurs while querying AWS resources
 export class ErrorNode extends AWSTreeNodeBase {
     public constructor(public readonly parent: AWSTreeNodeBase, public readonly error: Error, label: string) {
         super(label, vscode.TreeItemCollapsibleState.None)
+        const commandName: string = localize('AWS.command.viewLogs', 'View AWS Toolkit Logs')
+        const tooltip: string = localize('AWS.explorerNode.error.tooltip', 'Click to view error in Toolkit logs')
+        const logger: Logger = getLogger()
+        const logRecord: LogTrackerRecord = getLogTracker().registerLog()
```

Comment:
I wonder if the tracker should exist in the Logger. What's the rationale for having a separate global `getLogTracker()` that needs to be used in parallel? Would it make sense to instead expose it as `getLogger().registerLog()` ?

Reply:
The idea was to have 1 tracker per transport, not per logger. In this case we only are tracking a single log file. The `Logger` interface abstracts away the fact that it's implementations could be writing to multiple log files with different levels. The tracker could still be made apart of `Logger`, but it will need a way to identify transports that were added to Winston.

Reply:
Ok, see https://github.com/aws/aws-toolkit-vscode/pull/1569#discussion_r594759876 for a proposal